README.md

```diff
@@ -2,17 +2,17 @@
 
 ## Prepare release
 
-**Clone repository and checkout to branch 4.1**
+**Clone repository and checkout to branch v4.2.5**
 ```
 git clone https://github.com/wazuh/wazuh-bosh
 cd wazuh-bosh
-git checkout 4.1
+git checkout v4.2.5
```

Comment:
Could it be that there were additional steps for large files? or did that not fit here?

Reply:
The large files are now stored in S3 instead of Github. So, those steps were removed.